11/umbraco-cms/fundamentals/backoffice/settings.md

```diff
@@ -0,0 +1,83 @@
+---
+description: A guide displaying the options available in the Settings section in Umbraco Backoffice.
+---
+
+# Backoffice Settings Section
```

Comment:
Remember that this title needs to be the same as the title you give it in the navigation.

Also, as this article is only about Dashboards in the Settings section, I think the name/title should reflect that!

Reply:
I'll change the article name to "Dashboards in the Settings section"

Reply:
Of if that's too long, perhaps "Settings Dashboards"? 🤔